tests/acceptance/info-test.js

```diff
@@ -32,11 +33,6 @@ module('Info Tab', {
 });
 
 test("Libraries are displayed correctly", async function t(assert) {
-  let infoRoute = App.__container__.lookup('route:info');
```

Comment:
Per slack conversation, in glimmer2 ember the `service:-document` is not injected until an application instance is created with the first visit, so we cannot lookup a route like that.

With this workaround we don't need to.